pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -102,6 +102,16 @@ func (c *Cluster) configureDashboardModules() error {
 			hasChanged = true
 		}
 	}
+
+	ssoChanged, err := c.configureSSO()
+	if err != nil {
+		return errors.Wrap(err, "Failed to initialize SSO")
+	}
+
+	if ssoChanged {
+		logger.Info("SSO setup successful!")
+	}
```

Comment:
That SSO setup has been changed doesn't mean it was successful in case of disabling SSO, let's just remove it here and add a log line to the sso setup logic.